docs/root/intro/arch_overview/security/_include/ssl.yaml

```diff
@@ -34,9 +34,9 @@ static_resources:
         "@type": type.googleapis.com/envoy.extensions.transport_sockets.tls.v3.UpstreamTlsContext
         common_tls_context:
           tls_certificates:
-            certificate_chain: { "filename": "/cert.crt" }
-            private_key: { "filename": "/cert.key" }
-            ocsp_response: { "filename": "/ocsp_response.der" }
+            certificate_chain: { "filename": "certs/servercert.pem" }
+            private_key: { "filename": "certs/serverkey.pem" }
+            # ocsp_response: { "filename": "/ocsp_response.der" }
```

Comment:
i think this will fail validation anyway due to missing `der` file - but i guess it should be renamed to `ocsp_staple`

Reply:
nm, i pointed it to the `der` file in the test certs folder